RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Row/NonFungibleAssetRow+View.swift

```diff
@@ -70,14 +71,38 @@ extension NonFungibleAssetList.Row.View {
 			}
 			.padding(.horizontal, .medium1)
 			.padding(.top, .large3)
-			.padding(.bottom, .medium1)
-			.background(.app.white)
+			.padding(.bottom, .medium1 + CGFloat(viewStore.stackedCards) * .small2)
 		}
 	}
 
 	private var headerHeight: CGFloat { HitTargetSize.small.frame.height + 2 * .medium1 }
 }
 
+private extension ViewStoreOf<NonFungibleAssetList.Row> {
+	var stackedCards: Int {
+		if self.isExpanded {
+			0
+		} else if self.resource.nonFungibleIdsCount == 1 {
+			1
+		} else {
+			2
+		}
+	}
+
+	var rowBackground: some View {
+		Group {
```

Comment:
Is there a particular reason to not just annotate this with `@ViewBuilder` and skip the Group?

Reply:
no, same result